cr-core/build.gradle

```diff
@@ -26,8 +26,25 @@ targetCompatibility = 1.8
 
 // We use both Maven Central and our own Artifactory instance, which contains module builds, extra libs, and so on
 repositories {
-    mavenCentral()
+    mavenCentral {
+        content {
+            // This is first choice for most java dependencies, but assume we'll need to check our
+            // own repository for things from our own organization.
+            // (This is an optimization so gradle doesn't try to find our hundreds of modules in 3rd party repos)
+            excludeGroupByRegex("org.terasology(..+)?")
```

Comment:
this doesn't hurt, but it's not such a concern for this project that does not involve the many many terasology module dependencies

Reply:
I'm not only looking at this from a performance point, but also from a security point.
Any dependencies crafted to mimick dependencies we would expect to only be resolved by our artifactory and published from an external entity on public maven repositories would first be satisfied by the public one as it's lsited before our artifactory.